index.js

```diff
@@ -20,6 +20,7 @@ function all (list) {
   let size = 0
   for (let i = 0; i < list.length; i++) {
     const buf = list[i]
+    if (buf == null) continue
```

Comment:
can you move up the below condition also, ie

`buf === null || buf.buffer.byteLength === buf.byteLength`